pkg/operator/render.go

```diff
@@ -92,6 +92,7 @@ func createDiscoveredControllerConfigSpec(infra *configv1.Infrastructure, networ
 	}
 
 	ccSpec := &mcfgv1.ControllerConfigSpec{
+		NetworkType:         network.Status.NetworkType,
```

Comment:
Ah, you need to use network.Spec.NetworkType here, not status.  I don't think status is filled in until CNO comes up.

Reply:
Damn, alrighty, that’s an issue tho cause the api describes the status as the source of truth not the spec (the proxy populated the status at bootstrap). This is another point in having a CNO bootstrap command that is actually responsible for laying down these machineconfigs but yeah, I’ll fix this

Reply:
fixed, now using spec